samcli/hook_packages/terraform/hooks/prepare/makefile_generator.py

```diff
@@ -88,6 +89,12 @@ def generate_makefile(
     )
     shutil.copy(copy_terraform_built_artifacts_script_path, output_directory_path)
 
+    samcli_root_path = Path(os.path.dirname(__file__)).parent.parent.parent.parent
+
+    # copy zip.py script into output directory
+    ZIP_UTILS_MODULE_script_path = os.path.join(samcli_root_path, "local", "lambdafn", ZIP_UTILS_MODULE)
```

Comment:
nit: use all lowercase for variable name - https://peps.python.org/pep-0008/#function-and-variable-names